### PR TITLE
docs: Stop-gap info about filtering by stage, project,service

### PR DESCRIPTION
**Pull request by StackScribe**
This PR adds a bit of information about filtering by stage, project, or service.

This a stop-gap measure until we can create reference pages for the values.yaml file
and Distributor (see https://github.com/keptn/keptn.github.io/issues/1092 and https://github.com/keptn/keptn.github.io/issues/1094 )

@grabnerandi  Please check to see that I reproduced your words correctly.  In particular, confirm that these values are set in the values.yaml file and passed to the Distributor from there.  We can do a lot more when we do reference pages --

### CODE REVIEW

File: distributor/README.md [hunk 1]
```diff
@@ -1,28 +1,33 @@
 # Distributor
-
-A distributor queries event messages from NATS and sends the events to services that have a subscription to the event topic.
-Thus, each service has its own distributor that is configured by the two environment variables:
+A distributor subscribes a Keptn service with the Keptn Control Plane.
+More specifically, it queries event messages from NATS
```
[the remark this answers is not in the document]
Comment (StackScribe) [reply distributor/README.md]: THANK YOU!  This is exactly the sort of richness that we need in these docs!  See if you approve of my fix.

**Review by grabnerandi — APPROVED** [review]
lgtm

**Review by oleg-nenashev — APPROVED** [review]
All edits look good

**Review by oleg-nenashev — APPROVED** [review]
Will merge once the CI passes

**Pull request merged**